SteamKit2/SteamKit2/Steam/Handlers/SteamAuthentication/SteamAuthentication.cs

```diff
@@ -155,7 +157,7 @@ internal AuthSession( SteamUnifiedMessages.UnifiedService<IAuthentication> authe
             /// <returns>An object containing tokens which can be used to login to Steam.</returns>
             /// <exception cref="InvalidOperationException">Thrown when an invalid state occurs, such as no supported confirmation methods are available.</exception>
             /// <exception cref="AuthenticationException">Thrown when polling fails.</exception>
-            public async Task<AuthPollResult> StartPolling( CancellationToken? cancellationToken = null )
+            public async Task<AuthPollResult> PollingWaitForResultAsync( CancellationToken? cancellationToken = null )
```

Comment:
The standard convention is `CancellationToken cancellationToken = default`.

`default` is the same as `CancellationToken.None`.

This removes the need for null-checks further down.